### PR TITLE
Feature/enhancement

### PR DESCRIPTION
**Pull request by lianke123321**

### CODE REVIEW

File: centinel/primitives/traceroute.py [hunk 1]
```diff
@@ -119,20 +120,18 @@ def traceroute(domain, method="udp", cmd_arguments=None,
...
+                                "ip": probe.ip,
+                                "rtt": probe.rtt,
+                                "anno": probe.anno})
         hop_json["probes"] = probes_json
         hops.append(hop_json)
 
-    results = {}
     results["dest_name"] = parsed_output.dest_name
     results["dest_ip"] = parsed_output.dest_ip
-    results["method"] = method
```
Comment (rpanah) [inline centinel/primitives/traceroute.py]: @lianke123321 why did you remove this information from the results?

Comment (lianke123321) [inline centinel/primitives/traceroute.py]: Cause I found that in all circumstances "method" is always in results dict, so I removed redundant results["method"] = method and moved them to line 45 of the same file.

File: centinel/primitives/traceroute.py [hunk 2]
```diff
@@ -38,25 +42,27 @@ def traceroute(domain, method="udp", cmd_arguments=None,
     logging.debug("%sRunning traceroute for "
                   "%s using %s probes." % (log_prefix, domain, method))
 
+    results = {"method": method}
```
Comment (lianke123321) [inline centinel/primitives/traceroute.py]: @rpanah this is the new place where method is added.

**Comment by lianke123321** [comment]
@rpanah recommitted my changes based on new master branch
